sonar.bat

```diff
@@ -0,0 +1,30 @@
+@echo off
```

Comment:
Make sure you remove these, I just cleaned these up. Add them to the .gitignore if you have to